<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="user" failures="1" tests="1" name="Default test(failed)" time="0.05" errors="0" timestamp="2022-06-27T13:20:43 IST">
  <testcase classname="testNgKeyWords.MYgroupingClass1" name="a" time="0.01">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNgKeyWords.MYgroupingClass1.a(MYgroupingClass1.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- a -->
  <testcase classname="testNgKeyWords.MYgroupingClass1" name="d" time="0.0">
    <ignored/>
  </testcase> <!-- d -->
  <testcase classname="testNgKeyWords.MYgroupingClass1" name="c" time="0.0">
    <ignored/>
  </testcase> <!-- c -->
  <testcase classname="testNgKeyWords.MYgroupingClass1" name="b" time="0.0">
    <ignored/>
  </testcase> <!-- b -->
</testsuite> <!-- Default test(failed) -->
